README.md

```diff
@@ -74,62 +72,49 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Updated general bug report information.
 
-## New Spec - Feedback please!
+### Color Picker
 
-- What is new in PowerToys (SCOOBE) - [Pull Request](https://github.com/microsoft/PowerToys/pull/10978)
+- New fix to prevent the creation of duplicate colors in the selection history. Thanks [@DoctorNefario](https://github.com/DoctorNefario)!
+- Fixed OOBE hotkey description. Thanks [@coc0a](https://github.com/coc0a)!
+- Updated Color Picker GIF for OOBE. Thanks [@niels9001](https://github.com/niels9001)!
 
 ### FancyZones
-- Editor UX bug fixes. Thanks [@niels9001](https://github.com/niels9001)
-- Monitor resolution is added to the top to directly infer the boxes on top are your monitors
-- Fix for editor crash when editing a custom layout
 
-### PowerRename
-- Option added for capitalization.  
-- Improved loading responsiveness with large sums of files.
+- Full keyboard support added for the canvas editor’s main window and context. Thanks [@niels9001](https://github.com/niels9001)!
+    - Use `Arrows` to move a zone by 10 pixels or `Ctrl + Arrows` to move the zone by 1 pixel
+    - `Shift + Arrows` to resize a zone by 10 pixels (5 per edge), `Ctrl + Shift + Arrows` to resize a zone by 2 pixels (1 per edge)
+    - `Ctrl + Tab` to switch between the editor and dialog
+- New support for faster layout selection by double clicking a desired layout from the editor to automatically apply it and dismiss the editor.
+- New zone activation behavior allows users to snap a window to the zone who's center is closest to the cursor. Thanks [@ulazy1](https://github.com/ulazy1)!
+- Added process icon for FancyZones.
+- Fixed issue with zoning minimized windows.
 
 ### PowerToys Run
-- Changed XAML to improve rendering. Thanks [@niels9001](https://github.com/niels9001)
-- Disabled plugins are no longer loaded
-- VS Code plugin workspaces showing up now.  Thanks [@ricardosantos9521](https://github.com/ricardosantos9521)
-
-### Keyboard manager 
-- Now an independent exe.  This now runs high priority in its own process.  When your CPU is under load, this should allow the process to continue to be prioritized
-
-### Color Picker 
-- uses a centralized keyhook.  This should improve activation
-- Esc for closing will no longer bubble through.  Thanks [@DoctorNefario](https://github.com/DoctorNefario)
 
-### Settings / Welcome to PowerToys
-- Shortcuts will stand out more
-- Few accessability bugs fixed. Thanks [@niels9001](https://github.com/niels9001)
+- New Unit Converter plugin! Activate in the query prompt with the default activation phrase `%%`. Ex: `%% 10 ft in m`. Thanks [@jsoref](https://github.com/jsoref) and [@ThiefZero](https://github.com/ThiefZero)!
+- New Windows Settings plugin! Search for specific Windows settings from PowerToys Run by utilizing the default activation phrase `$` followed by the desired setting. Ex: `$ Add/Remove Programs`
```

Comment:
```suggestion
- New Windows Settings plugin! Search for specific Windows settings from PowerToys Run by utilizing the default activation phrase `$` followed by the desired setting. Ex: `$ Add/Remove Programs` or '$ Apps:'. Thanks [@ToniasSekan](https://github.com/TobiasSekan) and [@htcfreek](https://github.com/htcfreek).
```

- The second example show how to list all settings of an area category.
- It was a long time work and hard work to get this in. 😉

Reply:
@crutkas
Can we update readme to have at least both examples?

The `:` has a special function: Filter by area name only.

Reply:
Great call out @htcfreek ! I didn't know this functionality was available. We definitely want to make sure it gets recognized and documented, so thank you for point this out!